hello_world.py

```diff
@@ -7,6 +7,15 @@ def parse_args():
     parser.add_argument("name")
     return parser.parse_args()
 
+
+def add_value():
+  business = 'foo'
+  product = 'bar'
+  value = "{} makes {}".format(business, product)
+  return value
+
+
 if __name__ == "__main__":
     args = parse_args()
     print("Hello {}!".format(args.name))
+    print("Provided value in a theoretical world where we have this all figured out and things work just right: {}".format(add_value()))
```

Comment:
You know we aren't supposed to allow lines to get this long, what are you doing?